test/expected_dir/octave1.m

```diff
@@ -4,7 +4,7 @@
 octave	comment	## 3-clause BSD license appended to this file.
 octave	blank	
 octave	code	function varargout = toledolu(LU)
-octave	comment	  ## -*- texinfo -*-
+octave	comment	  ## (*- texinfo -*)
```

Comment:
Part of the issue with mathematica parsing was that when octave styled comments `#` were included within mathematica comments `(*...*)`, the file was detected as an octave file. We [moved around the conditional](https://github.com/blackducksoftware/ohcount/pull/49/files#diff-d77fa1db75cc45114696de9b1c005b26L791) a bit to give mathematica preference over octave.
This change to octave test file was to assert that in the reverse scenario where mathematica style comments `(*...*)` were included within octave comments `#`, octave files are still being detected correctly. Reverting back this change to octave test files will not affect our fix; please confirm and I will get it done right away.